nixos/modules/services/matrix/draupnir.nix

```diff
@@ -0,0 +1,278 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
```

Comment:
```suggestion
```
please don't use `with` like this.
to address #208242 for this module, uses of `with` such as this should be avoided.
consider `inherits` where you find yourself using `lib.foo` excessively.

Reply:
to clarify, just removing `with lib;` here wont do, consider using `inherits` to get from `lib.*`.
a lsp may be useful here as it will show you where you are trying to use something you don't have access to.